content/api/creating-superadmins-and-resellers.md

```diff
@@ -0,0 +1,36 @@
+# Creating Admin and Reseller Users
+
+## Create Super Admin Users
+
+Users with the super admin role can see a dashboard where they can see all server logs, teams, and users. They can make changes to all of these.
+
+Before inviting any employees consider setting up forceTwoFactor, onlyAllowDomain and ipRestrictions for your 'master' organization. 
+
+Invite the user to your organization. Once the user has created their account you can escalate them to admin permissions in SQL.
+
+```bash
+select * from users WHERE primaryEmail = 'employee@example.com';
```

Comment:
This wouldn't work because we store `primaryEmailId`, not `primaryEmail`. That ID is coming from the `emails` database, so maybe it's a good idea to skip directly to the user iD. We can say something like "Copy the ID of the user you want to escalate the permission for" and then continue to the `update` step.

Reply:
Apologies, I use a slightly different user table. Have updated to remove and reword.